components/src/core/components/Input/TreeSelect/tree-select-input.scss

```diff
@@ -0,0 +1,361 @@
+@import '../../../../styles';
```

Comment:
Most of these CSS are there in the select-input.scss file. Better to reuse it and only add the new checkbox related styles to a new file and import both.